$MirOS: ports/emulators/qemu/snapshot/patches/patch-hw_sh7750_c,v 1.1 2008/05/07 15:53:11 tg Exp $
--- hw/sh7750.c.orig	Wed Sep 23 19:01:08 2009
+++ hw/sh7750.c	Sat Oct 17 16:35:43 2009
@@ -208,6 +208,7 @@ static uint32_t sh7750_mem_readb(void *o
 	error_access("byte read", addr);
 	assert(0);
     }
+    return (0);		/* shut up gcc */
 }
 
 static uint32_t sh7750_mem_readw(void *opaque, target_phys_addr_t addr)
@@ -242,6 +243,7 @@ static uint32_t sh7750_mem_readw(void *o
 	error_access("word read", addr);
 	assert(0);
     }
+    return (0);		/* shut up gcc */
 }
 
 static uint32_t sh7750_mem_readl(void *opaque, target_phys_addr_t addr)
@@ -289,6 +291,7 @@ static uint32_t sh7750_mem_readl(void *o
 	error_access("long read", addr);
 	assert(0);
     }
+    return (0);		/* shut up gcc */
 }
 
 #define is_in_sdrmx(a, x) (a >= SH7750_SDMR ## x ## _A7 \
